src/components/RestrictedDocumentSelection/RestrictedDocumentSelection.tsx

```diff
@@ -47,11 +49,15 @@ export const RestrictedDocumentSelection = trackComponent(
     nextStep,
     documentSelection,
     countryFilter,
+    steps,
```

Comment:
Not sure if we should rely/pass `Steps` here, the `documentSelection` is the filter, if we populate that one with the config , it should work. That is how we filtered the list if config is passed on like this:
```
 "documentTypes": {
          "driving_licence": {
            "country": "ESP"
          },
          }
```
wouldn't break the standard set up? or workflow?

Reply:
thanks, noted 👍